src/main/java/jenkins/scm/api/SCMNavigatorDescriptor.java

```diff
@@ -84,7 +84,9 @@ public String getIconFilePathPattern() {
     /**
      * Creates a default navigator, if there should be one displayed.
      * May be interpreted by {@code jenkins.branch.SpecificDescriptor}.
-     * If returning non-null, you may also provide a {@code newInstanceDetail} view as per {@link TopLevelItemDescriptor}.
+     * If returning non-null, you may also provide a {@code newInstanceDetail} view as per {@link TopLevelItemDescriptor}
+     * and implement {@link SCMNavigatorDescriptor#getDescription()}.
```

Comment:
Why would you implement `getDescription` if providing a `newInstanceDetail`? That description would not be used, AFAICT.